response_operations_ui/views/logout.py

```diff
@@ -8,5 +10,7 @@
 @logout_bp.route('/')
 def logout():
     logout_user()
+    with suppress(KeyError):
```

Comment:
Can you put a comment on this saying that you're trying to delete something and if it's not there, we don't care

Reply:
If we know it may not be in the session, it would be clearer to check for it's existence first:
```
if 'messages_survey' in session:
    session.pop('messages_survey')
```
It can be done on one line but it's not as clear:
```
session.pop("messages_survey", None)
```